test/e2e/apimachinery/webhook.go

```diff
@@ -376,3 +409,114 @@ func cleanWebhookTest(f *framework.Framework) {
 	_ = client.CoreV1().ConfigMaps(skippedNamespaceName).Delete(disallowedConfigMapName, nil)
 	_ = client.CoreV1().Namespaces().Delete(skippedNamespaceName, nil)
 }
+
+// newCRDForAdmissionWebhookTest generates a CRD
+func newCRDForAdmissionWebhookTest() *apiextensionsv1beta1.CustomResourceDefinition {
+	return &apiextensionsv1beta1.CustomResourceDefinition{
+		ObjectMeta: metav1.ObjectMeta{Name: crdName + "s." + crdAPIGroup},
+		Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
+			Group:   crdAPIGroup,
+			Version: crdAPIVersion,
+			Names: apiextensionsv1beta1.CustomResourceDefinitionNames{
+				Plural:   crdName + "s",
+				Singular: crdName,
+				Kind:     crdName,
```

Comment:
capital

Reply:
Fixed. The testserver has the same problem: 
https://github.com/kubernetes/kubernetes/blob/81fa823a6ce2dcb95681ff4c0e4bc41ba3c5fd8d/staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/resources.go#L49-L52